composeApp/build.gradle.kts

```diff
@@ -41,6 +41,9 @@ kotlin {
             implementation(libs.decompose)
             implementation(libs.decompose.extensions.compose)
             implementation(libs.serialization.json)
+
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.0")
+            implementation("com.arkivanov.essenty:lifecycle-coroutines:2.0.0-alpha02")
```

Comment:
Please add this dependency to `./gradle/libs.versions.toml`.

Reply:
Seems like @OptIn(ExperimentalComposeLibrary::class) using for implementation(compose.components.resources) (?) is `The opt-in annotation is redundant: no matching experimental API is used`, maybe we can remove annotation?

Reply:
Yeah, that's no longer experimental, we can remove it.